### PR TITLE
added links to ApiKey handling and FAQ sections

### PR DESCRIPTION
**Pull request by clombardi**
## Short description
Re-enabled three links, added the needed configuration to specify the target for two of them.

## List of changes proposed in this pull request
- Re-enabled the link that points to the ApiKey handling in the new notification page (above the stepper) in PA.
- Added two links in the header of the contact ("recapiti") page for PF only, which point to specific items of the FAQ page in the landing webapp ("sito vetrina").
- Added three env variables in PF, which point to the landing site URL and the path to the items within its FAQ page which must be the target of the added links.

## How to test
Launch PA, go to new notification, click the "Chiavi API" link in the subtitle.
image

Launch PF, go to the contacts page, click in the links for "avvisi di avvenuta ricezione" and "di cortesia" in the subtitle
image

### CODE REVIEW

File: packages/pn-personagiuridica-webapp/src/utils/constants.ts [hunk 1]
```diff
@@ -14,6 +14,9 @@ export const PAGOPA_HELP_EMAIL = process.env.REACT_APP_PAGOPA_HELP_EMAIL;
 // PN-2029
 // export const PAYMENT_DISCLAIMER_URL = process.env.REACT_APP_PAYMENT_DISCLAIMER_URL;
 
+export const LANDING_SITE_URL = process.env.REACT_APP_LANDING_SITE_URL;
+export const FAQ_DIFFERENT_AMOUNTS_SUFFIX = process.env.REACT_APP_FAQ_DIFFERENT_AMOUNTS_SUFFIX;
```
Comment (ZorbaDimatteo) [inline packages/pn-personagiuridica-webapp/src/utils/constants.ts:18]: @clombardi same comment like the one for persona fisica webapp

**Review by ZorbaDimatteo — CHANGES_REQUESTED** [review]
@clombardi we could consider the construction of faq links as similar as the construction that we use to declare routes in our applications (through constants and methods).
Let's try to not overload env variables

File: packages/pn-personafisica-webapp/src/utils/constants.ts [hunk 2]
```diff
@@ -13,6 +13,11 @@ export const PAGOPA_HELP_EMAIL = process.env.REACT_APP_PAGOPA_HELP_EMAIL;
 // PN-2029
 // export const PAYMENT_DISCLAIMER_URL = process.env.REACT_APP_PAYMENT_DISCLAIMER_URL;
 
+export const LANDING_SITE_URL = process.env.REACT_APP_LANDING_SITE_URL;
+export const FAQ_DIFFERENT_AMOUNTS_SUFFIX = process.env.REACT_APP_FAQ_DIFFERENT_AMOUNTS_SUFFIX;
```
[the remark this answers is not in the document]
Comment (clombardi) [reply packages/pn-personafisica-webapp/src/utils/constants.ts]: the consts which refer to paths inside the landing site have been shifted to a new `src/navigation/externalRoutes.const.ts` file.

File: packages/pn-personagiuridica-webapp/src/utils/constants.ts (hunk 1 above)
Comment (clombardi) [reply packages/pn-personagiuridica-webapp/src/utils/constants.ts]: same as precedent comment

**Review by ZorbaDimatteo — APPROVED** [review]

**Pull request merged**